Architecture.md

```diff
@@ -71,6 +71,79 @@
 
 ***TODO: [GIT] how are the projects stored and how is authentication handled upon push***
 
+As all PaaS providers, PaaSTech needs to store the code of the projects created by its clients. This is done using a Git repository, which is created upon project creation. The repository is then exposed to the client, who can push its code to it.
+
+We chose to use Git to store the code of the projects, because it is the most used version control system, and is the most adapted to store code. Using it makes the integration seamless for users who are probably already using git.
+
+##### Creating and deleting repositories
+
+When a user initializes a new project, a new repository has to be created, it also has to be deleted when the project is deleted.
```

Comment:
```suggestion
When a Cl;ient initializes a new Project, a new Git repository has to be created; it also has to be deleted when the Project itself is deleted.
```

Reply:
fixed in : 1605b01ae29d29b16fac44effe7e065ddb570491